Dependencies/Inno/Examples/64Bit.iss

```diff
@@ -0,0 +1,33 @@
+; -- 64Bit.iss --
```

Comment:
I would suggest excluding this `Dependencies/Inno/Examples` folder from the repo, as we don't really need them.